code/controllers/subsystem/air.dm

```diff
@@ -535,7 +535,7 @@ SUBSYSTEM_DEF(air)
 		EG = new
 		EG.add_turf(src)
 
-	for (var/turf/open/ET in atmos_adjacent_turfs)
+	for (var/turf/open/ET as anything in atmos_adjacent_turfs)
```

Comment:
This does actually kinda need to be here, adjacencies aren't always rebuilt instantly, because of an attempted explosion optimization, so we can't be guaranteed of the typing of this list